arch/x86/kvm/vmx/pkvm/hyp/io_emulate.h

```diff
@@ -32,4 +32,7 @@ struct pkvm_pio_emul_table {
 	DECLARE_BITMAP(bitmap, PKVM_MAX_PIO_EMUL_NUM);
 };
 
+bool pio_access_valid(int size);
```

Comment:
do we need expose pio_access_valid?

Reply:
Sorry this is not needed, will change to static.

Reply:
Changed to static